### ISSUE TITLE
It would be nice if the *-element repositories had an element.html for mass import

### ISSUE THREAD

**Issue by krainboltgreene** [Polymer/docs#1278]
Basically the title.

I realize I can create my own `element.html` that does dependency checking for me, but realistically for prototyping it'd be nice to just simply that whole process.

In addition (And I can make another issue for this) it'd also be nice to have an PolymerElement/elements repository, that depends on all elements.

**Comment by addyosmani** [comment]
At one point we talked about including a 'meta' element that would import in everything inside a particular element collection but never ended up shipping it. I'm uncertain if there was a technical reason why or it just slipped through the cracks. @tjsavage may have some insight here.

**Comment by krainboltgreene** [comment]
I have to say, I'm not even bothered that it did. This thing is so beautiful, not only in style, but also architecture and design.

Plus the whole elements chart thing? Genius.

Let me know if I can help get this piece in, otherwise yeah.

**Comment by addyosmani** [comment]
cc @rictic as an FYI too

**Comment by rictic** [comment]
In 0.5 land we provided something like this: https://github.com/Polymer/paper-elements/blob/master/paper-elements.html

It's useful for convenience. On the other hand, it's something of a performance footgun, which might be why we're not providing it anymore. Or maybe we just never got around to it in the post 0.8 world.

**Comment by krainboltgreene** [comment]
Hmm, well is there a HTML import "compile" solution yet? Where I can automatically look at bower installs and build down to a elements.html of my own?

**Comment by rictic** [comment]
The simple compilation solution is the [vulcanize](https://github.com/Polymer/vulcanize/) tool. If you're adventurous, we've just landed integration into closure compiler, which can enable significant further size wins.

Neither of these will be able to look into your bower dependencies to build out their imports, as there's not always a 1:1 mapping from bower to elements that you can import. A number of repos include more than one HTML file to import.

**Comment by krainboltgreene** [comment]
Yeah, any concat tool would have to be knowledgable about `<link rel="import" ...>`. Hmm. I mean it could be done.
